test/DurableTask.SqlServer.AzureFunctions.Tests/MultiTenancyCoreScenarios.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace DurableTask.SqlServer.AzureFunctions.Tests
+{
+    using Xunit.Abstractions;
+
+    public class MultiTenancyCoreScenarios : CoreScenarios
```

Comment:
I'd rather not introduce an empty class. Can we instead make CoreScenarios a concrete class again but keep the WithoutMultiTenancyCoreScenarios.cs derivative?

Reply:
Done